spec/lita/handlers/locker_spec.rb

```diff
@@ -151,6 +167,17 @@
                                  'Alice (@alice)')
     end
 
+    it 'preserves the state of the queue when there is one' do
+      send_command('locker resource create foobar')
+      send_command('locker label create bazbat')
+      send_command('locker label add foobar to bazbat')
+      send_command('lock bazbat', as: alice)
+      send_command('lock bazbat', as: bob)
+      send_command('steal bazbat', as: charlie)
+      send_command('locker status bazbat')
+      expect(replies.last).to eq('Label: bazbat, state: locked, owner: Charlie, queued: Bob')
```

Comment:
Consider changing this message to be a more readable instead of a series of fields? My suggestion.

```
(lock) bazbat currently locked by Charlie. Next up: Bob
```